devicemaps.json

```diff
@@ -81,7 +82,14 @@
               }
             }
           }
-        ],
+        ]
+      },
+      "90500-0151-G11": {
+        "description": "Madrid - 6 ethernet, 1 lte",
+        "alias": {
+          "vendor": "Silicom",
+          "sku": "90500-0151-G"
```

Comment:
this looks incomplete

Reply:
Good catch, yea, these alisases should be `90500-0151-`